beets/dbcore/db.py

```diff
@@ -167,7 +180,7 @@ def update(self, values):
         for key, value in values.items():
             self[key] = value
 
-    def items(self):
+    def items(self) -> Generator[Tuple, None, None]:
```

Comment:
According to https://docs.python.org/3/library/typing.html#typing.Generator, this could also be written more concisely as `-> Iterable[Tuple[str, Any]]`.

Reply:
IMO it's better to type a generator as a generator, since it can have different uses to a more general iterable. IT's rare but people do call specific generator functions such as `next()`.

Reply:
Sure, in fact I'm not at all certain that I got these suggestions for `Iterator`, `Iterable`,  and `Generator` right. But the example that you mention seems to be about the distinction between the former two (`Iterator`  has `__next__` , `Iterable` only has `__iter__`). So, `from my understanding, Iterator`  would in fact be correct here, since not full generality of generators is allowed. `Generator[Tuple[..], None, None]`, seems to be synonymous, and I don't feel strongly about using either of the two expresssions.